activities/Story.activity/index.html

```diff
@@ -12,13 +12,13 @@
 	<link rel="stylesheet" media="screen and (device-width: 1200px) and (device-height: 900px)"
 		href="lib/sugar-web/graphics/css/sugar-200dpi.css">
 	<link rel="stylesheet" href="css/activity.css">
-	<link rel="stylesheet" href="css/bootstrap-tour-standalone.min.css">
+	<link rel="stylesheet" href="css/introjs.css">
 	<link rel="stylesheet" href="css/quill/quill.core.css">
 	<link rel="stylesheet" href="css/libnotify.css">
 	<script src="lib/vue.min.js"></script>
 	<script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
-	<script src="lib/jquery.min.js"></script>
-	<script src="lib/bootstrap-tour-standalone.min.js"></script>
+	// <script src="lib/jquery.min.js"></script>
```

Comment:
Do not leave commented line.
Plus it's not really commented because Javascript comments are not recognized in HTML.